Src/sounds.c

```diff
@@ -322,18 +322,18 @@ void playBeaconTune3()
 // 		fwdgt_counter_reload();
 // 		signaltimeout = 0;
 
-// 		if(eepromBuffer[i] == 255){
+// 		if(eepromBuffer.buffer[i] == 255){
```

Comment:
as i has changed, this commented out code is now incorrect. Maybe remove the commented code? or fix it to use the tune[] array?